src/constants/task.ts

```diff
@@ -211,3 +211,14 @@ export const taskStatusToCopy = {
   [TaskStatus.Unscheduled]: "Unscheduled",
   [TaskStatus.WillRun]: "Will Run",
 };
+
+// Represents background colors for task statuses
+export const mapColorToTaskUmbrellaStatus = {
+  [green.base]: taskStatusToCopy[TaskStatus.Succeeded],
```

Comment:
Actually can we just map this to a status? Instead of mapping it to the mapped result of a status.

Reply:
Also we already have a mapping for background colors to task statuses are they not sufficient? https://github.com/evergreen-ci/spruce/pull/817/files#diff-09686debf3a71d22bf34e4110a7bcc5e9b6cf98e9fcf99f949a134cebbabea17R171